bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -2203,12 +2150,12 @@ private int updateBookieIdInCookie(final String bookieId, final boolean useHostn
                         for (File dir : ledgerDirectories) {
                             newCookie.writeToDirectory(dir);
                         }
-                        LOG.info("Updated cookie file present in ledgerDirectories {}", ledgerDirectories);
+                        LOG.info("Updated cookie file present in ledgerDirectories {}", (Object) ledgerDirectories);
```

Comment:
I don't think you need to cast to `(Object)`. any reason why did you make this change?

Reply:
That's my reason, it can't be run in IDE because there has a warning and `-Werror` was set. I will recover it.